main.go

```diff
@@ -157,43 +226,70 @@ func main() {
 func setupRouter(d database.IDatabase) *gin.Engine {
 	log := log.New()
 	r := gin.New()
+
+	store, _ := sessions.NewRedisStore(10, "tcp", redisURL, "", []byte(os.Getenv("SECRET_KEY")))
```

Comment:
I think I'm a bit confused. With the use of JWT, do we need session at all anymore? I think we can drop it completely?

Reply:
Oh, I got you. I think you're right, I thought some of the Redis stuff might have been using it (I've never used it before).

Reply:
Naw, I started using redis to store sessions since I was using it for pub/sub other places. You can kill the session stuff completely (for this branch anyways).